app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Kong for Kubernetes deployment options
+---
+
+## Deployment options
+
+Kong for Kubernetes provides two options for its Kong Enterprise container
+image:
```

Comment:
The user doesn't have the necessary context here.
Before we jump into this section, we need a section to explain:

k4k8s consists of two components, a controller and a proxy. A diagram would help here.

Borrowing some content from the following pages could help:
https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/concepts/design.md
https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/concepts/deployment.md

Reply:
Don't recall offhand what the asset upload process is, but I think the prose explanation should suffice here: it's pretty short and simple.